### PR TITLE
git.md - put the GitHub token content in a collapsed section

### PR DESCRIPTION
**Pull request by mjbear**
git.md - put the GitHub token content in a collapsed section

### CODE REVIEW

**Comment by mthiel117** [comment]
Please change to not use inline HTML.  

Also, please use `mkdocs serve` to review your desired changes.

File: workshops/git.md [hunk 1]
```diff
@@ -280,7 +280,10 @@ If you do not have a GitHub account, you can create one **[here](https://github.
 
 ### **Create a GitHub personal access token**
 
-To push your local repo to GitHub, you will need a Personal Access Token. From your **GitHub** account, click through the following path to generate a new personal access token.  **Profile --> Settings --> Developer Settings --> Personal Access Tokens --> Tokens (classic) --> Generate new token (classic)**
+<details>
```
Comment (mthiel117) [inline workshops/git.md]: Please change to not use inline HTML.

Also, please use mkdocs serve to review your desired changes.

File: workshops/git.md [hunk 2]
```diff
@@ -280,7 +280,10 @@ If you do not have a GitHub account, you can create one **[here](https://github.
...
+    <summary>When not using the ATD lab IDE, generate a GitHub personal access token (click to expand)</summary>
```
Comment (mthiel117) [inline workshops/git.md]: Please change to not use inline HTML.

Also, please use mkdocs serve to review your desired changes.

**Review by mthiel117 — CHANGES_REQUESTED** [review]

**Comment by mjbear** [comment]
> Please change to not use inline HTML.
> 
> Also, please use `mkdocs serve` to review your desired changes.

Certainly.
I should have used an mkdocs [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) instead of the [general collapsible HTML markup](https://github.com/squidfunk/mkdocs-material/discussions/3711#discussioncomment-8225404). Apologies.

Will do.

**Comment by mjbear** [comment]
@mthiel117
This has been remedied by using an admonition for the collapsed section.

**Review by JulioPDX — APPROVED** [review]
Great add, thank you

**Comment by JulioPDX** [comment]
fixes #127

**Review by mthiel117 — APPROVED** [review]
LGTM.  I like it.  Thanks

**Pull request merged**